core/src/main/java/org/polypheny/db/adapter/Adapter.java

```diff
@@ -487,18 +262,23 @@ public void addInformationPhysicalNames() {
         );
         informationElements.add( physicalColumnNames );
 
-        Catalog catalog = Catalog.getInstance();
         group.setRefreshFunction( () -> {
             physicalColumnNames.reset();
-            List<CatalogPartitionPlacement> cpps = catalog.getPartitionPlacementsByAdapter( adapterId );
-            cpps.forEach( cpp ->
-                    catalog.getColumnPlacementsOnAdapterPerTable( adapterId, cpp.tableId ).forEach( placement -> {
-                        physicalColumnNames.addRow(
-                                placement.columnId,
-                                catalog.getColumn( placement.columnId ).name,
-                                cpp.physicalSchemaName + "." + cpp.physicalTableName + "." + placement.physicalColumnName );
-                    } )
-            );
+            List<PhysicalEntity> physicalsOnAdapter = new ArrayList<>();//snapshot.physical().getPhysicalsOnAdapter( adapterId );
+
+            for ( PhysicalEntity entity : physicalsOnAdapter ) {
```

Comment:
This does nothing because physicalsOnAdapter is always empty.

Reply:
Adjusted